tensorflow/python/ops/nn_ops.py

```diff
@@ -794,6 +794,71 @@ def _BatchNormGradShape(op):
 ops.RegisterShape("MaxPool")(common_shapes.max_pool_shape)
 
 
+@ops.RegisterShape("FusedResizeAndPadConv2D")
```

Comment:
maybe not worth it here, since the cpp shape function would be the final place - but is there overlap between this and the other conv ops?